src/jdk.hotspot.agent/bsd/native/libsaproc/dwarf.cpp

```diff
@@ -0,0 +1,329 @@
+/*
```

Comment:
Is the Dwarf implementation really OS specific, or should we be able to generalize this across all operating systems that uses Dwarf data?